pkg/utils/filehandler.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"github.com/gabriel-vasile/mimetype"
+)
+
+var SupportedFileTypes = []string{
```

Comment:
I think a common pattern here is to use a map[string]struct{}{} as a set, rather than needing to iterate through a list.